tekton/cronjobs/bases/image-build/trigger-image-build.yaml

```diff
@@ -83,4 +84,6 @@ spec:
                 value: "gcr.io/tekton-releases/dogfooding/myimage:latest"
               - name: PLATFORMS
                 value: ""
+              - name: BUILD_TYPE
+                value: ""
```

Comment:
NIT: perhaps we could have a default value of "docker" - but maybe it's better to require it to be explicit in the cronjob definition.

Reply:
By default we have single platform build - neither platforms nor buildType value

Reply:
Ah, right, good point, thank you